src/components/layouts/auth-wrapper/auth-wrapper.js

```diff
@@ -32,6 +32,8 @@ export const LoginLink = ({ label = "Login", className = "" }) => (
   <a
     className={className}
     href={`${BACKEND_URL}/oidc/login`}
+    target="_blank"
+    rel="noopener noreferrer"
```

Comment:
this link is for the login so we shouldn't include them here

Reply:
okay